CHANGELOG

```diff
@@ -1,3 +1,9 @@
+Version 0.4.1 (2020-04-03)
+--------------------------
+Repeater: fix log messages being attributed to ERROR level in log viever (#77)
+Repeater: provide control over insertion concurrency (#78)
```

Comment:
Just a nit and I know we've done it elsewhere but this is not a bugfix, it adds new functionality and ought to be a minor version bump according to semver rules.

Reply:
Sorry @dilyand - I don't think this is a nit. Version numbers are one of the most fundamental ways that software authors (here the Snowplow engineers) communicate with the consumers of software (here, the Snowplow community, customers, other teams (Tech Ops, Support etc)). As far as I know we are committed to using semver. Understanding what new software releases *do* is a fundamental aspect of software release and rollout. 

@goodits - can you take this back into the team and make sure we are following semver everywhere? I am disappointed to see we are still falling down on this...

Reply:
>  As far as I know we are committed to using semver. 

We are and it does follow [SemVer](https://semver.org/#spec-item-4):

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time.

Reply:
@alexanderdean and @dilyand are correct if, as I understand, the CLI command is changing. It's effectively our "public API", per:
MUST be incremented if new, backwards compatible functionality is introduced to the public API

We should get this changed before release. Alex raised a good point in a chat offline that the versions, when used correctly, are our public record. Appreciate we've trying to get this out to address a specific bug, but the value of the version shouldn't be underestimated. We'll be hanging other comms and processes off the version points. 

I'll raise this example with the team next week so we can make sure we're all on the same page.

Reply:
Sorry @chuwy we were writing at the same time.  I see how you're interpreting that. I think it does suggest a bit of free reign, but it would still make sense to bump the minor ver as an indicator that there's a new feature to leverage? If you could/should bump either, it would be minor? 

How I interpret that statement (we can look at other examples next week) is that if your change would otherwise meet the criteria of major (backward incompatibility) you shouldn't bump major unless stable. 

We can pick this up next week!

Reply:
In the end, the concurrency parameter didn't work out as I expected - I'll need to switch to a cached thread pool, so even this CLI option will go away (there's no thread capacity for cached TPs).

Reply:
Thanks guys. I doubt we'll land on the `0.x.x == anything goes at any time` definition - but if we did, then it would be incredibly important to get off the `0.x.x` version series quickly (certainly before components went into production) as the `0.x.x` version numbers would have dangerously low entropy.